core/src/main/java/com/gentics/mesh/changelog/highlevel/change/FixNodeVersionOrder.java

```diff
@@ -73,6 +74,7 @@ public String getDescription() {
 	public void apply() {
 		AtomicLong nodeCount = new AtomicLong();
 		AtomicLong fixedNodes = new AtomicLong();
+		NodeMigrationActionContextImpl context = new NodeMigrationActionContextImpl();
```

Comment:
```suggestion
		NodeMigrationActionContext context = new NodeMigrationActionContextImpl();
```

Reply:
The Impl is necessary here to access the conflicts later.